Sources/BraveShared/Extensions/BraveAdsExtensions.swift

```diff
@@ -6,11 +6,15 @@
 import Foundation
 import BraveCore
 import Shared
+import UIKit
 
 extension BraveAds {
   public func initialize(_ completion: @escaping (Bool) -> Void) {
     self.initialize(
       with: .init(),
+      sysInfoInfo: .init(
+        deviceId: UIDevice.current.identifierForVendor
+      ),
```

Comment:
This doesn't compile, 1. `with` is the sys info arg, 2. `identifierForVendor` returns an optional `UUID` not a string:

```diff
- with: .init(
+ with: .init(deviceId: UIDevice.current.identifierForVendor?.uuidString ?? ""),
```

Reply:
Fixed, thanks